.gitlab-ci.yml

```
@@ -55,7 +55,7 @@ variables:
   RUSTY_CACHIER_COMPRESSION_METHOD: zstd
   NEXTEST_FAILURE_OUTPUT:          immediate-final
   NEXTEST_SUCCESS_OUTPUT:          final
-  ZOMBIENET_IMAGE:                 "docker.io/paritytech/zombienet:v1.3.22"
+  ZOMBIENET_IMAGE:                 "docker.io/paritytech/zombienet:v1.3.30"
```

```suggestion
  ZOMBIENET_IMAGE:                 "docker.io/paritytech/zombienet:v1.3.33"
```

Update the zombienet version to the latest.